backend/src/main/java/com/huellapositiva/domain/actions/SubmitProposalRevisionAction.java

```diff
@@ -38,7 +38,8 @@ public void execute(String proposalId, ProposalRevisionDto revisionDto, URI prop
         ESAL esal = proposalRepository.fetch(proposalId).getEsal();
         ContactPerson contactPerson = contactPersonRepository.findByJoinedEsalId(esal.getId().toString());
         Reviser reviser = Reviser.from(credentialsRepository.findByEmail(revisionDto.getReviserEmail()));
-        if (revisionDto.getHasFeedback() == true && revisionDto.getFeedback() == null) {
+        Boolean hasFeedback =  revisionDto.getHasFeedback();
+        if (hasFeedback != null && hasFeedback && revisionDto.getFeedback() == null) {
```

Comment:
Why not use boolean?

Reply:
The sonarlint recomendation was that. What do you suggest?

Reply:
Sonar recommended to write the code in that way if the case you use a Boolean object, but is not telling you whether you should use a Boolean or a boolean. That is up to your requirements.